src/routers/messages.py

```
@@ -1,16 +1,21 @@
 from typing import Optional, List
 
-from fastapi import APIRouter, BackgroundTasks, Depends, HTTPException
+from fastapi import APIRouter, BackgroundTasks, Depends
 from fastapi_pagination import Page
 from fastapi_pagination.ext.sqlalchemy import paginate
 from sqlalchemy.sql import insert
 
 from src import crud, schemas
 from src.db import SessionLocal
 from src.dependencies import db
+from src.exceptions import ResourceNotFoundException, ValidationException
```

Ruff (0.8.2)

`src.exceptions.ValidationException` imported but unused

Remove unused import: `src.exceptions.ValidationException`

(F401)